src/Chapter1.hs

```diff
@@ -653,7 +663,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+    | abs n < 10 = n
```

Comment:
In case of `n` being negative, this would return negative result. as you are using `n` here 🙂 
But you are on the right way! 💪🏼

Reply:
Ouch! I ran `make test-chapter1-basic` only, and hence missed the failure - should have caught this :-/